src/huntsman/pocs/tests/test_huntsman.py

```diff
@@ -85,356 +85,3 @@ def pocs(config_with_simulated_stuff, observatory):
     yield pocs
 
     pocs.power_down()
-
```

Comment:
@wtgee The reason I deleted these is a) they are outdated and designed to test with the old config system and b) they seem to be mainly testing `pocs` functionality, rather than `huntsman-pocs`. Do we need this code hanging around?

Reply:
I don’t think so and would prefer it was removed if not needed. I just wanted to check.